R/plotly.R

```diff
@@ -119,6 +120,10 @@ plot_ly <- function(data = data.frame(), ..., type = NULL,
   id <- new_id()
   # avoid weird naming clashes
   plotlyVisDat <- data
+  # automatically namespace source
+  if (!is.null(session)) {
+    source <- session$ns(source)
```

Comment:
Why do you want the source namespaced? Don't you want to be able to have multiple modules share the same source?